### DISCUSSION TITLE
nginx inc ingress controller telemetry - 404 Not Found

### DISCUSSION THREAD

**Discussion by serifeturksever** [nginx/kubernetes-ingress#6740] [Q&A]
i am using nginx inc ingress controller in openhift. i configured otlp and telemetry ingress controller ymal as below

otlp:

kind: Ingress
apiVersion: networking.k8s.io/v1
metadata:
name: otlp
namespace: istio-system
annotations:
ingress.kubernetes.io/ssl-redirect: 'false'
kubernetes.io/ingress.class: nginx
nginx.org/basic-auth-realm: Authentication Required - otlp
nginx.org/basic-auth-secret: basic-auth-secret
spec:
tls:
- hosts: xxx
secretName: xxx
rules:
- host: xxx
http:
paths:
- path: /v1/traces
pathType: Prefix
backend:
service:
name: jaeger-collector
port:
number: 4318

zipkin:

kind: Ingress
apiVersion: networking.k8s.io/v1
metadata:
name: zipkin
namespace: istio-system
annotations:
ingress.kubernetes.io/ssl-redirect: 'false'
kubernetes.io/ingress.class: nginx
nginx.org/basic-auth-realm: Authentication Required - zipkin
nginx.org/basic-auth-secret: basic-auth-secret
nginx.org/rewrites: serviceName=jaeger-collector rewrite=/$1
spec:
tls:
- hosts: xxx
secretName: xxx
rules:
- host: xxx
http:
paths:
- path: /*(api/.*)
pathType: Prefix
backend:
service:
name: jaeger-collector
port:
number: 9411

otlp request worked correctly. But when i request telemetry addresses "https://xxx/api/v2/spans" return 404 Not Found. How can i fix this?

Nginx Ingress Operator version:
2.3.2 provided by NGINX Inc

actually path is in zippin like this:

![zipkin-path](https://github.com/user-attachments/assets/ca50c226-26b5-47b0-b9f1-8582f31223e4)

**Comment by vepatel** [comment]
duplicate of https://github.com/nginxinc/kubernetes-ingress/issues/6741
